k8s/services/pixie/helm-2.1.cue

```diff
@@ -0,0 +1,55 @@
+package pixie
+
+import (
+	"k8s.io/api/core/v1"
+	// "stakpak.dev/devx/k8s"
+)
+
+#KubeVersion: [=~"^2\\.1\\.6"]: minor: >=21
+#Values: [=~"^2\\.1\\.6"]: {
+
+    clusterRegistrationWaitImage: {
+        repository: "gcr.io/pixie-oss/pixie-dev-public/curl"
+        tag:        "1.0"
+        pullPolicy: v1.PullPolicy | *"IfNotPresent"
+    }
+
+    image: {
+        repository: "newrelic/newrelic-pixie-integration"
+        tag:        string | *"latest"
+        pullPolicy: v1.PullPolicy | *"IfNotPresent"
+        pullSecrets: []
+    }
+
+    resources: {
+        limits: {
+            memory: string | *"250M"
+        }
+        requests: {
+            cpu:    string | *"100m"
+            memory: string | *"250M"
+        }
+    }
+
+    podAnnotations: {}
+    podLabels: {}
+
+    job: {
+        annotations: {}
+        labels: {}
+    }
+
+    proxy: {}
+
+    nodeSelector: {}
+
+    tolerations: []
+
+    affinity: {}
+
+    customScripts: {}
+
+    excludeNamespacesRegex: ""
+    excludePodsRegex: ""
```

Comment:
Can you use types for this like `k8s.#annotations` or `v1.#affinity` you can find refernces to these types in other charts